packages/test/test-service-load/src/nodeStressTest.ts

```diff
@@ -36,36 +38,44 @@ async function main() {
     await orchestratorProcess(
             driver,
             { ...profile, name: profileArg },
-            { testId, debug, verbose });
+            { testId, debug, verbose, seed });
 }
 /**
  * Implementation of the orchestrator process. Returns the return code to exit the process with.
  */
 async function orchestratorProcess(
     driver: TestDriverTypes,
     profile: ILoadTestConfig & { name: string },
-    args: { testId?: string, debug?: true, verbose?: true },
+    args: { testId?: string, debug?: true, verbose?: true, seed?: number },
 ) {
-    const testDriver = await createTestDriver(driver, undefined);
+    const seed = args.seed ?? Date.now();
+
+    const testDriver = await createTestDriver(
+        driver,
+        seed,
+        undefined);
 
     // Create a new file if a testId wasn't provided
     const url = args.testId !== undefined
         ? await testDriver.createContainerUrl(args.testId)
-        : await initialize(testDriver);
+        : await initialize(testDriver, seed);
 
     const estRunningTimeMin = Math.floor(2 * profile.totalSendCount / (profile.opRatePerMin * profile.numClients));
-    console.log(`Connecting to ${args.testId ? "existing" : "new"} Container targeting with url:\n${url }`);
+    console.log(`Connecting to ${args.testId ? "existing" : "new"} with seed 0x${seed.toString(16)}`);
+    console.log(`Container targeting with url:\n${url }`);
     console.log(`Selected test profile: ${profile.name}`);
     console.log(`Estimated run time: ${estRunningTimeMin} minutes\n`);
 
-    const p: Promise<void>[] = [];
+    const runnerArgs: string[][] = [];
     for (let i = 0; i < profile.numClients; i++) {
         const childArgs: string[] = [
             "./dist/runner.js",
             "--driver", driver,
             "--profile", profile.name,
             "--runId", i.toString(),
-            "--url", url];
+            "--url", url,
+            "--seed", `0x${seed.toString(16)}`,
```

Comment:
Woah I didn't know you could pass in a numerical param as hex and NS would treat it as a number